openqaoa/optimizers/result.py

```diff
@@ -301,3 +303,46 @@ def lowest_cost_bitstrings(self, n_bitstrings: int = 1) -> dict:
             ]
         }
         return best_results
+
+
+    def dumps(self, as_string:bool=False, indent:int=4):
+        """
+        Returns the result serialized 
```

Comment:
maybe worth specifying -- "Returns serialized results for QAOA", if that's the case.